TheThingsAPI.class.nut

```diff
@@ -173,9 +223,17 @@ class TheThingsAPI {
         return function(resp) {
             // If we didn't get a 2xx status code, there was an error..
             if (resp.statuscode < 200 || resp.statuscode >= 300) {
+                if (resp.statuscode == 429) {
```

Comment:
The 429 was originally getting overwritten with a JSON decoding error. Looks like I fixed the JSON issue, so I will remove the special handling for the 429.